package.json

```diff
@@ -3,16 +3,19 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "start": "npm-run-all -p serve serve:css",
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "vue-cli-service test:e2e",
     "lint": "vue-cli-service lint --max-warnings 3",
     "pretty": "prettier --check '**/*'",
-    "test": "vue-cli-service test:unit && vue-cli-service test:e2e"
+    "test": "vue-cli-service test:unit && vue-cli-service test:e2e",
+    "serve:css": "postcss src/styles/tailwind.css -o src/styles/index.css"
```

Comment:
If you look at the Readme, most of these commands are described in there, in various sections. Do you think you could add something to the Readme to describe what `yarn run start` and `yarn run serve:css` will do? Thanks

Reply:
Done